pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -689,6 +690,39 @@ func toRule(r *v1beta.NetworkPolicyRule, policy *v1beta.NetworkPolicy, maxPriori
 	return rule
 }
 
+// toStretchedNetworkPolicySecurityRule converts v1beta.NetworkPolicyRule to *rule
+// which is only used for security purpose of StretchedNetworkPolicy. The security
+// rule arbitrarily drops all traffic initiated from Pods with UnknownLabelIdentity.
```

Comment:
Maybe remove "arbitrarily" if all traffic will be dropped for sure.

Reply:
Done.